lib/util/platform.js

```diff
@@ -645,6 +645,25 @@ shaka.util.Platform = class {
     return true;
   }
 
+  /**
+   * On some platforms, such as v1 Chromecasts, the act of seeking can take a
+   * significant amount of time.
+   *
+   * @return {boolean}
+   */
+  static isSeekingSlow() {
+    const Platform = shaka.util.Platform;
+    if (Platform.isChromecast()) {
+      if (Platform.isAndroidCastDevice()) {
```

Comment:
And Fuchsia?

Reply:
For reference...
I didn't have a Fuchsia device to test with, so I can't say for sure it's not a problem in them. It's probably fine, I was just being purposefully conservative with this change.